api/src/api/agencies_v1/agency_schema.py

```diff
@@ -39,3 +43,30 @@ class AgencyListResponseSchema(AbstractResponseSchema, PaginationMixinSchema):
         fields.Nested(AgencyResponseSchema),
         metadata={"description": "A list of agency records"},
     )
+
+
+class AgencyV1Schema(Schema):
+    agency_id = fields.UUID(
+        metadata={"description": "The internal ID of the agency", "example": "123res45"},
```

Comment:
The example you had wasn't a UUID - UUID has a default example already so we don't need to specify it generally.
```suggestion
        metadata={"description": "The internal ID of the agency"},
```